man/tpm2_policysecret.1.md

```diff
@@ -0,0 +1,75 @@
+% tpm2_policysecret(1) tpm2-tools | General Commands Manual
+%
+% OCTOBER 2018
+
+# NAME
+
+**tpm2_policysecret**(1) - Enables secret(password/hmac) based authorization to
+a policy.
+
+# SYNOPSIS
+
+**tpm2_policysecret** [*OPTIONS*]
+
+# DESCRIPTION
+
+**tpm2_policysecret**(1) Enables secret (password/hmac) based authorization to a
+ policy. The secret is the authentication value of any TPM object (NV/Hierarchy/
+Loaded/Persistent Handles).
```

Comment:
missing space indentation.

Reply:
Also describe the argument. As an argument, it takes the auth value of the associated TPM object, a single dash `-` can be used to read the auth value from stdin. The argument follows the "formatting standards for passwords ... (go find that verbatim text and drop it in here with the see that reference stuff)